src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const partToConvert = toFormat.slice(-1);
+  const partToFormat = toFormat.slice(0, 3);
+  const partfromFormat = fromFormat.slice(0, 3);
+  const splitDate = date.split(fromFormat[3]);
+  const arr = [];
+  const objDate = {};
+
+  for (const part of splitDate) {
```

Comment:
This loop is unnecessary as you can directly assign splitDate to arr using Array.from.